client/hostinfo_unix.cpp

```diff
@@ -77,7 +77,7 @@
 #include <sys/swap.h>
 #endif
 
-#if HAVE_SYS_SYSCTL_H
+#if HAVE_SYS_SYSCTL_H && !defined (__linux__)
```

Comment:
`HAVE_SYS_SYSCTL_H` is defined in `config.h` that is generated during `./configure` phase. I think it's better to put such check there instead of disabling it for all linux builds including old one